renovate.json

```diff
@@ -20,7 +20,8 @@
       "enabled": false,
       "groupName": "ignored python dependencies",
       "matchManagers": [
-        "setup-cfg"
+        "setup-cfg",
+        "pep621"
```

Comment:
We could remove "setup-cfg" nope ?

Reply:
I keep it to be consistent with what we made on joanie.

Reply:
In this commit https://github.com/openfun/joanie/commit/61e94e0a5c9f607ca949e476b22d56f330b243be

Reply:
Oh right !